RoddyCore/src/de/dkfz/roddy/config/ConfigurationIssue.groovy

```diff
@@ -54,8 +54,8 @@ class ConfigurationIssue {
         ),
         inproperVariableExpression(
                 ConfigurationIssueLevel.CVALUE_WARNING,
-                "Several variables in your configuration might misuse the variable import wrong. A variable needs to be imported into another value with \${variable identifier}.",
-                "The variable named '#REPLACE_0#' might misuse the variable import. Make sure, that all intended variable imports are non-escaped and formed like \${variable identifier}. Also, variable nesting is not allowed."
+                "Several variables in your configuration might use malformatted variable imports. Variable imports must look like \${variable identifier}, nesting is forbidden.",
```

Comment:
1. "Several". One is sufficient. Just "Variable(s)"
2. "might" expresses extreme uncertainty. Use "may" if you are really uncertain. Use "are" if you are certain.